### DISCUSSION TITLE
关于「团队」内容由 `列表` 转换为 `图片表格` 的讨论

### DISCUSSION THREAD

**Discussion by hzpt-inet** [hzpt-inet-club/inet.org#11] [General]
将从
![image](https://user-images.githubusercontent.com/88970987/151914072-777b032f-c28f-49cd-9735-69a493ffb918.png)
转变成为
![image](https://user-images.githubusercontent.com/88970987/151914094-f3ad1c64-76ec-48a3-a382-ff1dd2351ae5.png)
进行讨论。

**Comment by elonehoo** [comment]
## 结论

我觉的这样确实比「列表要展示的信息量更大」

## 理由

因为图片确实比文字更有看点。
